src/js/off-canvas.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+/*global document window*/
+
+(function() {
+
+  const html = document.querySelector('html');
+  const header = document.querySelector('.site-header');
+  const menu = document.querySelector('.site-header__menu');
+  const menu_open = document.querySelector('.site-header__menu-open');
+  const menu_close = document.querySelector('.site-header__menu-close');
+
+  const media_query = window.matchMedia('(width < 64em)')
+
+  function toggleMenu() {
+    menu_open.addEventListener('click', (e) => {
```

Comment:
Suggestion (non-blocking): We could maybe pick these event handler functions out and refactor this a little to make it easier to also unbind the click events (should the user resize from small to large) 🤙

Reply:
Cool, let's revisit this in a refactor JS ticket.